README.md

```diff
@@ -27,13 +29,67 @@ For general project information, visit the [Harmony wiki](https://wiki.earthdata
 5. [Contributing to Harmony](#Contributing-to-Harmony)
 6. [Additional Resources](#Additional-Resources)
 
+## Minimum System Requirements
+
+* A running [Docker Desktop](https://www.docker.com/products/developer-tools) or daemon instance - Used to invoke docker-based services
+* A running [Kubernetes]() cluster. [Docker Desktop](https://www.docker.com/products/docker-desktop) for Mac and Windows comes with a
+built-in Kubernetes cluster which can be enabled in preferences.
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) - A command-line application for interfacing with a Kubenetes API.
+
+## Quick-ish Start (Mac OS X / Linux)
+
+If you are interested in using a local Harmony instance to develop services, but not interested in 
+developing the Harmony code itself, the following steps are enough to start a locally running Harmony instance.
+
+1. Follow the directions for creating an Earth Data Login application and credentials in the [Earthdata Login Application Requirement](#Earthdata-Login-Application-Requirement) section below.
+
+2. Download this repository
+```bash
+git clone https://github.com/nasa/harmony.git
+```
+3. Clone the example service repository (or any other Harmony service) into the `services` sub-directory
+```bash
+cd harmony/services && git clone https://github.com/nasa/harmony-service-example
+```
+4. Go to the `harmony` directory and build the service
+```bash
+cd .. && ./bin/build-service-images
+```
+5. Generate a `.env` file
+```bash
+./bin/create-dotenv
+```
+6. Fill in the generated `.env` file with your EDL OAUTH credentials from step 1
```

Comment:
New abbreviations.  Also it may be worth somewhere mentioning that they can contact a Harmony dev if they have trouble with this step, i.e. so we can just give them the values.

Reply:
Do we have generic values that we can give out?

Reply:
I added a blurb to the script to tell people to contact the Harmony team if they need help with EDL